powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/callbacks/CallbackUrlEventService.java

```diff
@@ -62,7 +62,7 @@ public class CallbackUrlEventService {
 
     private final CallbackUrlEventRepository callbackUrlEventRepository;
     private final CallbackUrlEventResponseHandler callbackUrlEventResponseHandler;
-    private final LoadingCache<String, CachedRestClient> restClientCache;
+    private final LoadingCache<String, CachedRestClient> callbackUrlRestClientCache;
```

Comment:
If we are not sending the events - because of the circuit breaker, we need to treat it as WARN. INFO is acceptable, but debug simply means we will have no logs about events that were not dispatched.

logger.debug("Callback URL has reached failure threshold, associated events are not dispatched: callbackUrlId={}", event.getCallbackUrlEntity().getId());